src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -208,7 +209,10 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
           RawlsGroupMemberList(userSubjectIds = Some(subjectIds))
         }}
       }
-      case Some(id) => Future.successful(RawlsGroupMemberList(userSubjectIds = Some(Seq(id))))
+      case Some((fcUser, nihUser)) =>
+        if(whitelist contains nihUser) Future.successful(RawlsGroupMemberList(userSubjectIds = Some(Seq(fcUser))))
+        else Future.failed(throw new FireCloudExceptionWithErrorReport(ErrorReport(StatusCodes.Forbidden,
```

Comment:
technically, the junk is not inaccurate. The inaccuracy is the statement "your access will be granted shortly"

under what conditions does it say no access?